package.json

```diff
@@ -55,7 +55,7 @@
 		"bytebuffer": "^5.0.1",
 		"color2k": "1.2.4",
 		"commonmark": "git+https://github.com/RocketChat/commonmark.js.git",
-		"commonmark-react-renderer": "git+https://github.com/RocketChat/commonmark-react-renderer.git",
+		"commonmark-react-renderer": "git+https://github.com/RocketChat/commonmark-react-renderer.git#improve/remove-edited-indicator",
```

Comment:
If I understood correctly, the right thing would be to merge this PR: https://github.com/RocketChat/commonmark-react-renderer/pull/1
And don't point to branch like you did here.
But I could be wrong...
Because when this PR is merged this branch will be deleted.

Reply:
@diegolmello